packages/javalib.2.3.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.1.1"
+name: "javalib"
+maintainer: "sawja@inria.fr"
+version: "2.3.1"
```

Comment:
I'd recommend updating the `opam-version` to `1.2` (it will be rewritten automatically for older clients using the HTTP interface to access the repository). Also, I recommend running `opam lint` on the files.